test/muleCI/Jenkinsfile

```diff
@@ -1,3 +1,3 @@
-@Library('go-algorand-ci') _
+@Library('go-algorand-ci@brice/commentOutDarwin') _
```

Comment:
I don't like custom branch builds; can we merge go-algorand-ci and then update this?

Reply:
Sure, I'll make a PR for that branch

Reply:
https://github.com/algorand/go-algorand-ci/pull/98

Reply:
This is updated to use master now